docs/mobile-sdk.md

```diff
@@ -1,7 +1,940 @@
+
+#### 2️⃣ Activating the SDK
+
+To activate the SDK you must use the `activate` function with your `projectId` (see the [Prerequisites](#prerequisites) section above to get that identifier). You have access to callbacks in order to know if the SDK has been successfully activated. In case of an SDK activation failure the callback will give you the reason of the failure and you may want to retry later:
+
+##### iOS
+<pre class="prettyprint lang-swift">
+IAdvizeSDK.shared.activate(projectId: 0000,
+                           authenticationOption: .simple(userId: "UserIdentifier"),
+                           gdprOption: .disabled)) { success in
+    if success {
+        ...
+    }
+}
+</pre>
+ 
+⌨️ **In-context example:** [SDK Activation](https://github.com/iadvize/iadvize-ios-sdk/blob/master/Example/IAdvizeSwiftExample/IAdvizeSwiftExample/Source/AppDelegate%2BiAdvize.swift#L61)
+
+##### Android
+
+On Android there is an additional step required before activating the SDK. You will need to provide a reference to your application object and initiate the SDK with it.
+
+In your `AndroidManifest.xml` declare your application class:
+
+<pre class="prettyprint lang-xml">
+&lt;application
+  android:name=&quot;my.app.package.App&quot;&gt;
+  &lt;!-- your activities etc... --&gt;
+&lt;/application&gt;
+</pre>
+
+This class should then initiate the SDK:
+
+<pre class="prettyprint lang-kotlin">
+package my.app.package.App
+    
+class App : Application() {
+  override fun onCreate() {
+    super.onCreate()
+    IAdvizeSDK.initiate(this)
+  }
+}
+</pre>
+
+> *⚠️ If you are developing an application using an hybrid framework based on a webview, you will need to do this call to `initiate` before the main webview activity is created, that means in the native Android app code, and not from the hybrid framework plugin. As the SDK relies on observing activities lifecycles to show/hide the chat button, not doing so will result in chat button not showing when it should.*
+
+After that you can activate the SDK:
+
+<pre class="prettyprint lang-kotlin">
+IAdvizeSDK.activate(
+  projectId = 0000,
+  authenticationOption = AuthenticationOption.Simple("UserIdentifier"),
+  gdprOption = GDPROption.Disabled,
+  callback = object : IAdvizeSDKCallback {
+    override fun onSuccess() {
+      Log.d("iAdvize SDK", "The SDK has been activated.")
+    }
+    override fun onFailure(t: Throwable) {
+      Log.e("iAdvize SDK", "The SDK activation failed with:", t)
+    }
+  }
+)
+</pre>
+
+⌨️ **In-context example:**
+
+- [SDK Initiation](https://github.com/iadvize/iadvize-android-sdk/blob/d15e3415c4d4a0afc097cfcfee67f50aa4631556/sample/app/src/main/java/com/iadvize/conversation/sdk/demo/App.kt#L40)
+- [SDK Activation](https://github.com/iadvize/iadvize-android-sdk/blob/master/sample/app/src/main/java/com/iadvize/conversation/sdk/demo/activities/MainActivity.kt#L143)
+
```

Comment:
Je pense qu'il faut qu'on rajoute une sous-section ici (avant le `you can choose between...) : 

`##### Authentication modes`